hoomd/Integrator.h

```diff
@@ -130,7 +130,7 @@ class PYBIND11_EXPORT Integrator : public Updater
             }
 
         /// Count the total number of degrees of freedom removed by all constraint forces
-        Scalar getNDOFRemoved(std::shared_ptr<ParticleGroup> query);
+        virtual Scalar getNDOFRemoved(std::shared_ptr<ParticleGroup> query);
```

Comment:
I may have missed something, but I don't think this method needed to be declared virtual for this PR.